src/devices/thermostat.cpp

```diff
@@ -593,12 +593,12 @@ void Thermostat::process_RC20Timer(std::shared_ptr<const Telegram> telegram) {
         uint8_t time = telegram->message_data[1];
 
         // we use EN settings for the day abbreviation
-        std::string sday = (FL_(enum_dayOfWeek)[day][0]);
```

Comment:
I've been doing the same, everytime there's a FL_ try and use const char *. It really releases a lot of heap memory. My free available went from 140kb to 153kb with a few changes I made over the weekend.